### PR TITLE
Remove information route and add a temporary redirect to /about for SEO purposes 

### PR DESCRIPTION
**Pull request by Ridhwana**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
/p/information is redundant since these links appear in the the sidebar and footer in any case. Hence, we made a decision to remove the links. However, after chatting with Ben, we decided that we will add a temporary redirect to /about for some time for SEO purposes. 

## Related Tickets & Documents
https://github.com/forem/InternalProjectPlanning/issues/228

## QA Instructions, Screenshots, Recordings
Go to /p/information and expect a redirect to the about page.

## Added tests?

- [ ] Yes
- [x] No, as I only removed a view.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] Docs.forem.com
- [ ] README
- [x] No documentation needed

## [optional] Are there any post deployment tasks we need to perform?
None 

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by rhymes — APPROVED** [review]
🔥

**Review by benhalpern — APPROVED** [review]

**Comment by benhalpern** [comment]
Looks like there's some stuff caught in regression tests.

File: config/routes.rb [hunk 1]
```diff
@@ -427,8 +427,12 @@
 
     get "/page/:slug" => "pages#show"
 
+    # TODO: [forem/teamsmash] removed the /p/information view and added a redirect for SEO purposes.
+    # We need to remove this route in 2 months (11 January 2021).
```
Comment (mstruve) [inline config/routes.rb:431]: Why 2 months?

**Review by mstruve — APPROVED** [review]
small question for my own curiosity otherwise LGTM!

File: config/routes.rb (hunk 1 above)
Comment (benhalpern) [reply config/routes.rb:431]: I think this is pretty arbitrary. Enough time to let stuff stop linking to it, not so much time to be a big deal if eventually there are broken links in a few months.

Comment (Ridhwana) [reply config/routes.rb:431]: sorry @mstruve just seen this now. What Ben said :)

**Pull request merged**